workflow-templates/GraphQL_Apollo_K8S_Demo.yml

```diff
@@ -0,0 +1,122 @@
+name: DEMO Build Workflow
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - development
+      - StandAloneGithubAction
+    paths-ignore:
+      - '**.md'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env: 
+        AZURE_REGISTRY_NAME: crdsmlint5b5869c1
+        CRDS_ENV: demo
+        VAULT_URI: https://vault.crossroads.net:443
+        VAULT_ROLE_ID: ${{secrets.VAULT_ROLE_ID}}
+        VAULT_SECRET_ID: ${{secrets.VAULT_SECRET_ID}}
+        DOCKER_TAG: ${{github.sha}}
+        ASPNETCORE_ENVIRONMENT: Demo
+        MP_OAUTH_BASE_URL: https://admindemo.crossroads.net/ministryplatformapi/oauth
+        MP_REST_API_ENDPOINT: https://admindemo.crossroads.net/ministryplatformapi
+        CRDS_MP_COMMON_CLIENT_ID: CRDS.Common
+        CRDS_MP_COMMON_CLIENT_SECRET:  ${{secrets.CRDS_MP_COMMON_CLIENT_SECRET}}
+        DOCKER_IMAGE_SECRET: some-value-that-nobody-should-know-487374
+        NEW_RELIC_APP_NAME: crds-graphql-groups
+        NEW_RELIC_DISTRIBUTED_TRACING_ENABLED: true
+        NEW_RELIC_LICENSE_KEY: ${{secrets.NEW_RELIC_LICENSE_KEY}}
+
+        REPO: crds-graphql-groups
+        GRAPH: groups

```

Comment:
I think we should make this an environment var. That way we can easily pick this template, then set the env and not have to change the action file. Just like we did with the .net k8s templates

Reply:
I just updated the values